src/util/kaldi-table-inl.h

```diff
@@ -1549,6 +1624,8 @@ class RandomAccessTableReaderScriptImpl:
     last_found_ = 0;
     script_.clear();
     current_key_ = "";
+    range_ = "";
+    data_rxfilename_last_ = "";
```

Comment:
there are actually many cases of using this in this header file. Should I change all of them to clear()?